examples/forms.py

```diff
@@ -2,13 +2,26 @@
 from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
 from django.contrib.auth.models import User
 
+from bootstrap_modal_forms.forms import BSModalModelForm, BSModalForm
 from bootstrap_modal_forms.mixins import PopRequestMixin, CreateUpdateAjaxMixin
-from bootstrap_modal_forms.forms import BSModalForm
-
 from .models import Book
 
 
-class BookForm(BSModalForm):
+class SimpleModalForm(BSModalForm):
```

Comment:
Please remove SimpleForm at all places. I think that FilterForm you implemented is awesome.

Reply:
done